modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with condition type, informing you of potential impacts to upgrading your cluster. 
+The following condition types are consistent across all ClusterVersion and ClusterOperator conditions. Individual components may also have additional condition types with component-specific language.
+
+The condition type, as well as additional informatin about your operator can be retrieved in either YAML or JSON format through the `oc get clusterversion -o` command:
```

Comment:
nit: "informatin" -> "information", and possibly a detour-ending comma: "..., as well as ... your operator, can be"?